### ISSUE TITLE
the red highlight (when pinged) is ugly

### ISSUE THREAD

**Issue by Fil** [vector-im/vector-web#441]
currently we have
```
.mx_EventTile_highlight {
    color: #ff0064;
```

about any other color might be better: red on white is not super readable, and does not seem to belong to the text flow

(sorry for the stupid issue)

**Comment by ara4n** [comment]
yup - we have a better design for this that needs to be implemented; looks like this:

<img width="311" alt="screen shot 2015-11-29 at 21 49 29" src="https://cloud.githubusercontent.com/assets/1294269/11460358/19ebfbe8-96e3-11e5-992e-37f0c829175e.png">

**Comment by turt2live** [comment]
The design above was seemingly before all kinds of other changes, like e2ee (as the lock icon would conflict with the mention bar). Would something like the following being just as desirable?
![image](https://user-images.githubusercontent.com/1190097/31475955-f73f5ed0-aec0-11e7-9361-8c64286ef081.png)

Personally I find the red/pink to be okay, although having the whole message highlighted feels weird.

**Comment by aaronraimist** [comment]
Related: #1655 #6520

**Comment by jtrees** [comment]
This is in the [v1 milestone](https://github.com/vector-im/riot-web/milestone/3) that has since been closed. Any chance it can be put back on the current roadmap? It fits [the accessibility milestone](https://github.com/vector-im/riot-web/milestone/37) pretty well too.
